intg-denonavr/avr.py

```diff
@@ -652,13 +650,6 @@ async def power_off(self) -> ucapi.StatusCodes:
         if not self._use_telnet:
             self._set_expected_state(States.OFF)
 
-    @async_handle_denonlib_errors
-    async def power_toggle(self) -> ucapi.StatusCodes:
-        """Send power-on or -off command to AVR based on current power state."""
-        if self._receiver.power is not None and self._receiver.power == "ON":
-            await self.power_off()
-        await self.power_on()
-
```

Comment:
This method is still used!
https://github.com/unfoldedcircle/integration-denonavr/blob/main/intg-denonavr/media_player.py#L247-L248